CorpusSearch/ClientApp/src/routes/Home.tsx

```diff
@@ -150,20 +150,26 @@ export const Home = () => {
             {hasNoSearch && stats && <span className={"homeText"}>
                 {stats != "error" ?
                     <span className={"homeText"} style={{textAlign: "center"}}>
-                        <span style={{display: "inline"}}>Search our growing collection of over <b title={`${stats.uniqueManxWordCount.toLocaleString()} unique words`}>{stats.manxWordCount.toLocaleString()} Manx words</b> or&nbsp;<a href={"/Browse"}>browse&nbsp;{stats.documentCount.toLocaleString()} documents</a></span>
-                    </span>    
-                :
+                        <span style={{display: "inline"}}>Search our growing collection of over <b
+                            title={`${stats.uniqueManxWordCount.toLocaleString()} unique words`}>{stats.manxWordCount.toLocaleString()} Manx words</b> or&nbsp;
```

Comment:
revert the auto-formatting changes.

If they're necessary, they should go in their own commit

Reply:
Okay

Reply:
Done